parsers/php.c

```diff
@@ -409,15 +412,20 @@ static tokenInfo *newToken (void)
 {
 	tokenInfo *const token = xMalloc (1, tokenInfo);
 
+	token->string = vStringNew ();
+	token->scope  = vStringNew ();
+
+	return token;
+}
+
+static void clearToken (tokenInfo *const token)
+{
 	token->type			= TOKEN_UNDEFINED;
 	token->keyword		= KEYWORD_NONE;
-	token->string		= vStringNew ();
-	token->scope		= vStringNew ();
-	token->lineNumber   = getInputLineNumber ();
```

Comment:
line number seems to have got lost in translation

…although, if it didn't break the test suite, maybe it's not so useful?